pkg/azurefile/azure.go

```diff
@@ -47,6 +48,32 @@ var (
 	storageService = "Microsoft.Storage"
 )
 
+func getRuntimeClassForPod(ctx context.Context, kubeconfig, podName string, podNameSpace string) string {
+	var (
+		kubeClient *clientset.Clientset
+	)
+	kubeCfg, err := getKubeConfig(kubeconfig, false)
+	if err == nil && kubeCfg != nil {
```

Comment:
We should probably return an error here if `err != nil` and handle it in the caller.